maestro-domain/src/test/resources/fixtures/DefaultIndexerTest/MALY-DE.conflicting.analysis.json

```diff
@@ -1,106 +1,98 @@
 [
   {
-    "analysisType": { "name": "variantCall", "version": 1 },
-    
+    "analysisType": {
```

Comment:
this is no longer valid right ? it was broken down to two fields

Reply:
It is valid, this is song payload, which is still the old format. https://github.com/icgc-argo/argo-metadata-schemas/pull/27/files